cloud_chatops/src/pr_reminder.py

```diff
@@ -1,175 +1,168 @@
 """This module handles the posting of messages to Slack using the Slack SDK WebClient class."""
 
-from typing import List, Dict
+from typing import List
 from datetime import datetime, timedelta
 from slack_sdk import WebClient
 from slack_sdk.errors import SlackApiError
-from src.read_data import get_token, get_user_map, get_repos
-from src.get_github_prs import GetGitHubPRs
-from src.pr_dataclass import PrData
+from read_data import get_token, get_user_map, get_repos
+from get_github_prs import GetGitHubPRs
+from pr_dataclass import PrData
+from custom_exceptions import ChannelNotFound
 
 
 class PostPRsToSlack:
+    # pylint: disable=R0903
+    # Disabling this as there only needs to be one entry point.
     """
     This class handles the Slack posting.
     """
 
-    def __init__(self):
-        super().__init__()
-        self.repos = get_repos()
-        self.client = WebClient(token=get_token("SLACK_BOT_TOKEN"))
+    def __init__(self, mention=False):
+        self.channel = "C06U37Y02R4"  # STFC-cloud: dev-chatops
```

Comment:
you should probably make this a CONSTANT_VARIABLE and define it at the top or better yet have it as a parameter in the constructor set to this as default. This allows us to change it more easily